src/index.html

```diff
@@ -11,6 +11,28 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="big-search-bar"
+    data-qa="big"
+    action=""
+    method="get">
+      <input
+        type="search"
+        class="search search-bar-1"
+        placeholder="Try &ldquo;Los Angeles&rdquo;"
+        data-qa="keypress"
+      >
+    </form>
+    <form
+    class="small-search-bar"
+    data-qa="small"
+    action=""
+    method="get">
```

Comment:
```suggestion
    <form
      class="small-search-bar"
      data-qa="small"
      action="#"
      method="get"
    >
```
Форматируй теги таким образом, и не оставляй атрибуты без значений